scan

```diff
@@ -240,7 +250,9 @@ def scan_domains(scanners, domains, options):
         # Kick off workers in parallel. Returns when all are done.
         scan_start_time = utils.local_now()
         with ThreadPoolExecutor(max_workers=workers) as executor:
-            tasks = ((scanner, domain, handles, environment, options) for domain in domains_from(domains))
+            tasks = ((scanner, domain, handles, environment, options)
+                     for domain in domains_from(
+                         domains, domain_suffix=options.get("suffix")))
```

Comment:
I don't want to artificially line-wrap lines this way, and I think they're already exempted in flake8 checks. While this might make things easier in certain code editors, this makes the code less readable overall.

Reply:
I'm pretty sure I wrapped that because it's over 100 characters wide, which I think is what's set in our flake8 config.
I'm not sure I agree on readability, but I'm very used to working on files limited to being 80 characters wide.